app/design/adminhtml/Magento/backend/Magento_Customer/web/css/source/_module.less

```diff
@@ -28,8 +28,8 @@
     }
 
     .customer-default-address-wrapper {
-        align-items: flex-start;
-        display: flex;
+        .lib-vendor-box-align(flex-start);
+        .lib-vendor-prefix-display(flex);
         float: left;
```

Comment:
default param .lib-vendor-prefix-display() is flex so you don't need to more specific except you want to try inline flex such as .lib-vendor-prefix-display(inline-flex)